### ISSUE TITLE
Training with data that has more than 3 channels

### ISSUE THREAD

**Issue by ne1114** [junyanz/pytorch-CycleGAN-and-pix2pix#1344]
Hello I'm trying to train with data that are more than 3 channels (ex 6, 12).

After the training I noticed that the model hasn't learned sufficiently compared to 3 channel data. (The 3, 6, 12 channeled datasets have images of interest stacked **channel wise**. Which means its basically grayscale image stacked 3 times vs. 6 times. 12 times.) 

I think it would be hard for the model fully capture the importance of **channel wise** just with L1 loss, as I stack more channels. 

Would there be any suggestions other than changing the loss function? For example, increase the `ngf, ndf`,  stack more network layers, etc  please?

**Comment by junyanz** [comment]
I am not sure if changing hyper-parameters will help your case. Depending on the type of data you are working on, you might need a new reconstruction loss rather than the default L1 loss.

**Comment by tongfh1** [comment]
Hello, I would like to ask you how did you implement the training of more than 3 channels of data, and which part of the code was modified?
